app/javascript/components/record-form/form/subforms/subform-fields/components/violation-item/styles.css

```diff
@@ -0,0 +1,18 @@
+
+.verifyFormControlRoot {
+  padding: 4px;
+}
+
+.verifySelectComponent {
+  padding: 12px !important;
+  border-radius: 4px;
+  border-bottom: none;
+}
+
+.verifySelectComponentSelect, .verifySelectComponentSelect:focus {
+  border-radius: 4px;
+}
+
+.verifySelectComponent[aria-expanded="true"], .verifySelectComponent:focus {
+  outline: 4px solid rgba(31, 176, 214, 0.4);
```

Comment:
The color used here should be added to our root css vars or use one of the existing css vars

Reply:
This color is a little odd. Could you attach a screenshot of this?